### ISSUE TITLE
Invalid value provided for 'state' error

### ISSUE THREAD

**Issue by luckymike** [hw-labs/knife-cloudformation#13]
```
FATAL: Unexpected Error: Invalid value provided for `state` (:update_complete_cleanup_in_progress). Allowed - :create_complete, :create_in_progress, :create_failed, :delete_complete, :delete_in_progress, :delete_failed, :rollback_complete, :rollback_in_progress, :rollback_failed, :update_complete, :update_in_progress, :update_failed, :update_rollback_complete, :update_rollback_in_progress, :update_rollback_failed
```

**Comment by chrisroberts** [comment]
https://github.com/miasma-rb/miasma/issues/8

**Comment by chrisroberts** [comment]
fixed via miasma release v0.2.16
